routes/web.php

```diff
@@ -13,6 +19,34 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
+Route::get('/', function(){
+  return view('stockapp');
+});
+Route::get('/stockapp',[IndexController::class,'index'])->name('stockapp');
+Route::get('/categories',function(){ return view('categories');})->name('categories');
+Route::get('/stores',function(){ return view('stores');})->name('stores');
+Route::get('/products',function(){ return view('products');})->name('products');
+
+Route::get('/contact',function(){return view('contact');})->name('contact');
+Route::get("/contact/create",[ContactController::class,'create'])->name('create');
+Route::post("/contact",[ContactController::class,'store'])->name('contact-create');
+
+Auth::routes();
+
+Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+
+
+Route::group(['prefix'=>'admin'],function(){
+
+  Route::get('/dashboard',[AdminController::class,'index'])->name('admin.dashboard');
+  Route::get('/profile',[AdminController::class,'profile'])->name('admin.profile');
+});
+
+  Route::group(['prefix'=>'user'],function(){
+  Route::get('/dashboard',[UserController::class,'index'])->name('user.dashboard');
+  Route::get('/profile',[UserController::class,'profile'])->name('user.profile');
+  Route::get('/reset/password',function(){return view('reset');})->name('reset');
```

Comment:
sen niye reset e yönlendiriyorsun? bunun için laravel authentication'ın route u yok mu zaten ?

Eğer bilmiyorsan veya unuttuysan. php artisan route:list yazarak route lara bakabilirsin. Ayrıca laravel authentication'ın dokümantasyonuna da bakabilirsin.